### DISCUSSION TITLE
Changes to php-fpm possible in AIO?

### DISCUSSION THREAD

**Discussion by Fred-DTV** [nextcloud/all-in-one#2218] [Questions] [answered]
Hi @szaimen 

I have a weird issue which wouldn't let me login anymore without clearing my cache beforehand.
After some searching I have found this post which says it could be a problem with the php-fpm ownership settings:
https://help.nextcloud.com/t/solved-cant-login-via-web-ui-anymore/31483

Because it says in the official NC manual that you can tweak the php-fpm settings I was already looking for it a couple o f weeks ago but couldn't find it in my docker install.
https://docs.nextcloud.com/server/16/admin_manual/installation/server_tuning.html#tune-php-fpm

Is editing of this file actually useful in AIO? If yes, is it located somewhere else?
Or is my problem not related to the settings in php-fpm at all?

Thanks again

**Comment by Zoey2936** [comment, marked as answer]
I can't help you with your Issue, but I can say that it is not possible for users to permanently modify the php configuration

**Reply by szaimen** [reply]
Indeed

**Reply by szaimen** [reply]
But https://github.com/nextcloud/all-in-one/discussions/2218#discussioncomment-5433389

**Reply by owopeachy** [reply]
Why not? How would we tweak it for performance reasons then?

**Reply by szaimen** [reply]
PHP-FPM is tweaked in regard to performance for you automatically so no need to do it manually.

See https://github.com/nextcloud/all-in-one/discussions/1335

**Comment by szaimen** [comment]
@Fred-DTV please refrain from pinging me directly. I'll answer as soon as I find time.

**Reply by Fred-DTV** [reply]
oh i'm so sorry, that was absolutely not meant to get your attention. I forgot that this will notifiy you directly... 
I am really thankful for your help and didn't want to disturb you on a sunday... Sorry

**Comment by szaimen** [comment]
> Or is my problem not related to the settings in php-fpm at all?

Yes, I think so
